notifier/orb.yaml

```diff
@@ -1,12 +1,77 @@
 version: 2.1
 description: "Tools for notifying external services about CI events"
 
+orbs:
+  slack: circleci/slack@4
+
 executors:
   alpine-stable:
     docker:
     - image: alpine:latest
     resource_class: small
 
+# This command has two conditions:
+# - Since it could be too noisy, it MUST be used only on production deploys!
+# - It MUST be used as a post-step from a job
+commands:
+  slack-on-fail:
+    description: |
```

Comment:
pro-tip: if you use `>` instead of `|` here, it's a signal to yaml to remove linebreaks in your string. That would let you keep this long description wrapped to a more readable limit (eg. 80 chars or such) without affecting the formatting. See [this description](https://github.com/talkiq/circleci-orbs/blob/master/gcloud/orb.yaml#L11-L13), for example.

For more info on this, [this](https://yaml-multiline.info/) is a great page.

Reply:
I changed it accordingly. Thanks Kev for this pro-tip. I was wondering by looking at other circleci files if `>` allows multiple lines. Now, I know it. :D